cmd/hamctl/command/rollback.go

```diff
@@ -48,7 +48,7 @@ has no effect.`,
 					return err
 				}
 
-				if len(releasesResponse.Releases) < 2 {
+				if len(releasesResponse.Releases) < 2 { //should there just be 1 release to do rollback? so len(releasesResponse.Releases) < 1?
```

Comment:
WDUT?

Reply:
What do you roll back to if you are rolling back the only release? 🤔

Reply:
Maybe I assume wrong but is the current release all part of the rollback options?